.pre-commit-config.yaml

```diff
@@ -1,4 +1,17 @@
 repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: check-added-large-files
+    -   id: check-byte-order-marker
+    -   id: check-merge-conflict
+    -   id: check-symlinks
+    -   id: check-toml
+    -   id: check-xml
+    -   id: detect-private-key
```

Comment:
I find this one a bit ironic, since that's part of what this tool itself detects 😄 Since we're running tartufo against itself, is this additional check necessary?